src/Widgets/ButtonWidget/ButtonWidgetEditingConfig.js

```diff
@@ -12,6 +12,7 @@ Scrivito.provideEditingConfig("ButtonWidget", {
         { value: "left", title: "Left" },
         { value: "center", title: "Center" },
         { value: "right", title: "Right" },
+        { value: "block", title: "Block level" },
```

Comment:
* [ ] FYI In the Bootstrap docs there is a technical term and a human readable term:
> block level buttons—those that span the full width of a parent

I think we should prefer the technical term on the tech side and the human term on the editor side:
```ts
{ value: "block", title: "Full width" },
```